modules/rdma-prerequisites.adoc

```diff
@@ -0,0 +1,12 @@
+// Module included in the following assemblies:
+//
+// * hardware_accelerators/rdma-remote-direct-memory-access.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="rdma-prerequisites_{context}"]
+
+= NVIDIA GPUDirect RDMA prerequisites
+
+All methods of NVIDIA GPUDirect RDMA configuration require the installation of specific Operators.
+Use the following steps to install the Operators.
```

Comment:
Similar to my other comment about removing the `.Next steps` title, I think ending the last sentence in a colon might make a more logical flow into the list that follows this. Up to you though
```suggestion
Use the following steps to install the Operators:
```

Reply:
Changed.